src/Util/TestListenerTrait.php

```diff
@@ -43,8 +47,11 @@ public function startTestSuite($mainSuite)
                 continue;
             }
             $testedClass = new \ReflectionClass($m[1].$m[2]);
-            $bootstrap = new \SplFileObject(\dirname($testedClass->getFileName()).'/bootstrap.php');
-            $warnings = array();
```

Comment:
this line seems to be a bug, the array `$warnings` gets cleared on every iteration, and its used after the loop. so i have moved it before the foreach, which i think makes more sense.

Reply:
good catch, fixed in https://github.com/symfony/polyfill/pull/315